### PR TITLE
[INJIWEB-662] : modify the issuers display properties for country demo

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: esignet-insurance.properties [hunk 1]
```diff
@@ -477,7 +477,7 @@ mosip.esignet.vci.key-values={\
                          }},\
                         'display': {{'name': 'Health Insurance', \
                                     'locale': 'en', \
-                                    'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png', 'alt_text': 'a square logo of a Veridonia'},\
+                                    'logo': {'url': 'https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png', 'alt_text': 'a square logo of a Veridonia'},\
```
Comment (swatigoel) [inline esignet-insurance.properties]: `alt_text` should be changed to tell it's stay protected insurance logo

File: esignet-insurance.properties [hunk 2]
```diff
@@ -505,7 +505,7 @@ mosip.esignet.vci.key-values={\
                        }},\
                       'display': {{'name': 'Life Insurance', \
                                     'locale': 'en', \
-                                    'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                    'logo': {'url': 'https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png','alt_text': 'a square logo of a Veridonia'},\
```
Comment (swatigoel) [inline esignet-insurance.properties]: `alt_text` should be changed to tell it's stay protected insurance logo

File: esignet-insurance.properties [hunk 3]
```diff
@@ -538,7 +538,7 @@ mosip.esignet.vci.key-values={\
                      }},\
                     'display': {{'name': 'Health Insurance', \
                                   'locale': 'en', \
-                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                  'logo': {'url': 'https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png','alt_text': 'a square logo of a Veridonia'},\
```
Comment (swatigoel) [inline esignet-insurance.properties]: `alt_text` should be changed to tell it's stay protected insurance logo

File: esignet-insurance.properties [hunk 4]
```diff
@@ -565,7 +565,7 @@ mosip.esignet.vci.key-values={\
                        }},\
                       'display': {{'name': 'Life Insurance', \
                                     'locale': 'en', \
-                                    'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                    'logo': {'url': 'https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png','alt_text': 'a square logo of a Veridonia'},\
```
Comment (swatigoel) [inline esignet-insurance.properties]: `alt_text` should be changed to tell it's stay protected insurance logo

File: mimoto-issuers-config.json [hunk 5]
```diff
@@ -151,64 +151,64 @@
...
-            "url": "https://${mosip.api.public.host}/inji/veridonia-logo.png",
-            "alt_text": "a Veridonia logo"
+            "url": "https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png",
+            "alt_text": "a square logo of a StayProtected"
           },
           "language": "en",
-          "title": "Veridonia Insurance Company",
-          "description": "Download Insurance Credentials using policy details"
+          "title": "Download StayProtected Insurance Credentials",
+          "description": "Download insurance credential"
```
Comment (swatigoel) [inline mimoto-issuers-config.json:161]: title and description looking similar, Download credential
should we change title to `StayProtected Insurance` or `StayProtected Insurance Department`

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: mimoto-issuers-config.json (hunk 5 above)
Comment (vijay151096) [reply mimoto-issuers-config.json:161]: Modified to "StayProtected Insurance" in title which will be the title in the mobile, similar to name which will be title in the web.

File: esignet-insurance.properties (hunk 4 above)
Comment (vijay151096) [reply esignet-insurance.properties]: Modified

File: esignet-insurance.properties (hunk 3 above)
Comment (vijay151096) [reply esignet-insurance.properties]: Modified

File: esignet-insurance.properties (hunk 2 above)
Comment (vijay151096) [reply esignet-insurance.properties]: Modified

File: esignet-insurance.properties (hunk 1 above)
Comment (vijay151096) [reply esignet-insurance.properties]: Modified

**Review by swatigoel — APPROVED** [review]

**Review by Rakshitha650 — APPROVED** [review]

**Pull request merged**